test/hook.afterPackageCreateDeploymentArtifacts.test.js

```diff
@@ -74,4 +74,149 @@ describe('Serverless warmup plugin after:deploy:deploy hook', () => {
 
     expect(fs.remove).not.toHaveBeenCalled()
   })
+
+  it('Should have default global package property', async () => {
+    const mockProvider = { request: jest.fn(() => Promise.resolve()) }
+    const serverless = getServerlessConfig({
+      getProvider () { return mockProvider },
+      service: {
+        custom: {
+          warmup: {
+            enabled: true
+          }
+        },
+        functions: { someFunc1: { name: 'someFunc1' }, someFunc2: { name: 'someFunc2' } }
+      }
+    })
+    const plugin = new WarmUp(serverless, {})
+
+    await plugin.hooks['after:package:initialize']()
+
+    expect(plugin.serverless.service.functions.warmUpPlugin.package)
+      .toEqual({
+        individually: true,
+        exclude: ['**'],
+        include: ['_warmup/**']
+      })
+  })
+})
+
+it('Should override global package property', async () => {
```

Comment:
Maybe just for consistency, this should be `Should use the package exclusions from options if present`.

Nothing major, please just take a look if the names match the rest of the tests.